.pre-commit-config.yaml

```diff
@@ -128,13 +128,13 @@ repos:
     hooks:
       - id: yamllint
   - repo: https://github.com/python-jsonschema/check-jsonschema
-    rev: 0.33.2
+    rev: 0.35.0
     hooks:
       - id: check-dependabot
       - id: check-github-workflows
   # TODO: get zizmor to pass, and maybe set it up as a github action
   # - repo: https://github.com/woodruffw/zizmor-pre-commit
-  #   rev: v1.11.0
+  #   rev: v1.18.0
   #   hooks:
   #     - id: zizmor
```

Comment:
I know that this is outside the scope of the publishing in general, but I just wanted to note that this is a TODO lying around outside the diff of this PR, and it'll help secure the publishing process when resolved. I'm happy to take this up separately if you'd like me to!